source/includes/concept/_dataset.md

```diff
@@ -0,0 +1,61 @@
+# Concepts
+
+## Dataset
+
+One of the Resource Watch API's (RW API) goals is to provide a common interface for interacting with data hosted in different sources and formats. A **Dataset** is the Resource Watch's API way of providing users with access to data, while trying to, as much as possible, abstract and standardise operations that would otherwise fall on the user's hands to figure out. It's one of the cornerstone onto which many other API resources build upon.
+
+**Example**: *A Resource Watch API dataset can represent data in a JSON file, hosted on [Carto](https://carto.com/) or [Google Earth Engine](https://earthengine.google.com/), to name a few. However, when accessing that data, you don't have to learn 3 different technologies - the Resource Watch API gives you a single, unified query interface.*
+
+
+On top of **datasets**, the RW API offers multiple resources that allow you to access data in multiple formats. These will be covered later in full detail, but as an example, here are some ways in which you can access and use datasets:
+
+* you can create **Layers** to display dataset's geographical information on maps;
+* you can create **Widgets**, graphic representations of dataset's data, which you can use, for instance, to build charts;
+* you can create **Subscriptions** associated with datasets and be notified via email of significant updates;
+* or you can perform your own **Queries** using a SQL-like syntax, and use the dataset data to build your own custom visualizations.
+
+
+### Dataset providers
+
+Each dataset has a **provider** (json/carto/GEE/...) that must specified on creation - it's this value that tells the RW API how to handle different data providers and formats, so you don't have to. Below you'll find a list of the different providers supported:
+
+
+#### Carto
+
+Carto is an open, powerful, and intuitive map platform for discovering and predicting key insights underlying the location data in our world.
```

Comment:
Carto is Open Source, but it's a payed service, might be worth mentioning that. Pricing details: https://carto.com/pricing/

Reply:
I agree with linking to Carto, but not necessarily saying that it's paid or anything. I'll add a "if you want to learn more about carto, click here (+link)" type of solution